docker/README.md

```diff
@@ -34,58 +33,112 @@ If you do not have or want to download the Orion repository, you can create a fi
 	  ports:
 	    - "1026:1026"
 	  command: -dbhost mongo
+```
+> Uncomment the `--smallfiles` line if you host doesn't have too much free space.
```

Comment:
This should be aligned with the .yml file we provide after changes in PR https://github.com/telefonicaid/fiware-orion/pull/1395, which uses (without comment):

`command: --smallfiles --nojournal`

The rationale is that it is better to use the "safest" configuration to run in every system (e.g. the ones which small disk space of slow). It doesn't hurt to newbie users and advanced users would know that this is about and can remove that commands if they don't want to use it.

This text could be used to explain a little bit what they are (maybe with a better wording):

"Regarding --smallfiles, you can ommit it if your hosting machine have enough disk space. Regarding --nojournal it is not recommened for production, but it speeds up mongo container start up and avoid some "race conditions" problems if Orion containers is faster and doesn't find the DB up and ready".